lib/common/exceptions/minor_codes.rb

```diff
@@ -8,6 +8,13 @@ module MinorCodes
         status: MajorCodes::UNPROCESSABLE_ENTITY
       }.freeze
 
+      CLIENT_ERROR = {
+        type: :client,
+        code: '900',
```

Comment:
Nit, but at some point in the near future can we get these back to being in numeric order?